### PR TITLE
Task seeds

### PR DESCRIPTION
**Pull request by Ayasunsun**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,81 @@
...
+hayato = User.new(params)
+hayato.save
+puts "Created user #{hayato.id}"
+
+leaders = [hayato, linlu]
+
+params = {}
+params[:name] = "Davide Zannet"
+params[:leader] = false
+params[:leader_id] = leaders.sample.id
```
[the remark this answers is not in the document]
Comment (Ayasunsun) [reply db/seeds.rb:31]: Thanks @davide32123 ! I will change them.

**Comment by Ayasunsun** [comment]
I fixed user seed on "user_branch" and created task seeds on this branch.

**Comment by Ayasunsun** [comment]
This branch doesn't have user seeds so this branch can't be merged until "user seeds" branch gets merged! I messed up.

File: db/seeds.rb [hunk 2]
```diff
@@ -48,3 +47,140 @@
   puts "Created patient #{new_patient.id}" if new_patient.save
 end
 puts "Finished creating patients"
+
+puts "creating tasks"
+params = {}
+params[:description] = "VS check"
+params[:duration] = "10mins"
```
Comment (davide32123) [inline db/seeds.rb:54]: duration is integer, please remove "min" part

**Comment by davide32123** [comment]
@JupitersLB can you double check the spelling of the tasks just in case?

File: db/seeds.rb [hunk 3]
```diff
@@ -48,3 +47,140 @@
...
+new_task = Task.new(params)
+new_task.save
+puts "Created task #{new_task.id}"
+
+params = {}
+params[:description] ="BS check"
```
Comment (JupitersLB) [inline db/seeds.rb:60]: Just to check this is supposed to BS and not VS?

File: db/seeds.rb [hunk 4]
```diff
@@ -48,3 +47,140 @@
...
+
+params = {}
+params[:description] = "Discharge"
+params[:duration] = "30mins"
+new_task = Task.new(params)
+new_task.save
+puts "Created task #{new_task.id}"
+
+params = {}
+params[:description] = "Diper"
```
Comment (JupitersLB) [inline db/seeds.rb:144]: diaper

File: db/seeds.rb [hunk 5]
```diff
@@ -48,3 +47,140 @@
...
+
+params = {}
+params[:description] = "Walk"
+params[:duration] = "20mins"
+new_task = Task.new(params)
+new_task.save
+puts "Created task #{new_task.id}"
+
+params = {}
+params[:description] = "Wund care"
```
Comment (JupitersLB) [inline db/seeds.rb:179]: wound

**Pull request merged**